docs/source/release_notes/release-notes.rst

```diff
@@ -43,7 +43,14 @@ develop
          - Change
 
     *    - |new|
-         - Limit number of concurrent Lock calls from the same client.
+         - The lock server now can limit number of concurrent connections from the same client which request a lock.
+           This behavior can be enabled with the configuration flag ``useThreadPooling`` on the ``TimeLockServerConfiguration`` in dropwizard.
+
+           The threads from the server are split equally in a `private pool` and a `public pool`.
```

Comment:
Lines 49-52 is probably TMI for a release note, and should instead go somewhere on [this page](https://palantir.github.io/atlasdb/html/configuration/external_timelock_service_configs/timelock_server_config.html).